cmd/deploy/deploy_test.go

```diff
@@ -480,7 +480,11 @@ func TestCreateConfigMapWithBuildError(t *testing.T) {
 	assert.Equal(t, expectedCfg.Name, cfg.Name)
 	assert.Equal(t, expectedCfg.Namespace, cfg.Namespace)
 	assert.Equal(t, expectedCfg.Labels, cfg.Labels)
-	assert.Equal(t, expectedCfg.Data, cfg.Data)
+
+	keysToCompare := []string{"actionName", "name", "status", "filename", "icon", "yaml"}
```

Comment:
Are they empty? We should check what's the UI is doing if we send the key but empty. I remember that we have a bug in the past because of that. Probably we need to only send them if they are not empty

Reply:
@jLopezbarb they were empty before, now they are populated but it's a side effect because the path in the test is the okteto/okteto repo so it detects the current branch used in okteto/okteto